operator/src/main/resources/application.properties

```diff
@@ -5,5 +5,5 @@ quarkus.container-image.builder=jib
 quarkus.operator-sdk.crd.validate=false
 
 # Operator config
-operator.keycloak.image=quay.io/keycloak/keycloak:latest
+operator.keycloak.image.name=quay.io/keycloak/keycloak
 operator.keycloak.image-pull-policy=Always
```

Comment:
One more thing. I think we could also move the pull policy under image to keep it consistent, i.e. `operator.keycloak.image.pull-policy`. WDYT?

Reply:
Mixed feelings, I feel like `image-pull-policy` reflects best the property in Kubernetes (which is a separate field `imagePullPolicy`), but not really fussy about it.